apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -59,6 +59,7 @@ export default function ListBoxInline({ app, fieldIdentifier, stateName = '$', o
     sortByState = 1,
     scrollState = undefined,
     setCount = undefined,
+    shouldConfirmOnBlur = undefined,
```

Comment:
Maybe it's just my personal preference, but I think it feels nicer calling it just `confirmOnBlur`.

Reply:
or actually I just merged the other PR so easier to stick with this

Reply:
that's fine, just a detail